src/grt/grt-wave_options-parse.ads

```diff
@@ -0,0 +1,64 @@
+--  GHDL Run Time (GRT) - mono-thread version.
+--  Copyright (C) 2005 - 2014 Tristan Gingold
```

Comment:
Fixed

By the way, I made some renaming to improve readability :

Grt.Wave_Options -> Grt.Wave_Opt_File
Grt.Wave_Opt_File.Read -> Grt.Wave_Opt_File.Tree_Reading